lib/rules/no-trailing-spaces.js

```diff
@@ -85,8 +93,8 @@ module.exports = {
 		/**
 		 * Report the error message
 		 * @param {ASTNode} node node to report
-		 * @param {int[]} location range information
-		 * @param {int[]} fixRange Range based on the whole program
+		 * @param {SourceLocation[]} location range information
+		 * @param {SourceRange[]} fixRange Range based on the whole program
```

Comment:
```suggestion
		 * @param {SourceLocation} location range information
		 * @param {SourceRange} fixRange Range based on the whole program
```

`SourceRange` is a tuple of elements with indices 0 and 1. That is probably why it was defined as an array of `int[]`: https://github.com/eslint/rewrite/blob/core-v0.13.0/packages/core/src/types.ts#L71-L74. Same in other files.

The previous line looks like a copy-paste error, but it's clear from the usage that this should be a `SourceLocation` and not an array.

Reply:
Gotcha.